modules/react-components/src/components/documentation/documentation-link.tsx

```diff
@@ -58,21 +63,25 @@ export const DocumentationLink: FunctionComponent<PropsWithChildren<Documentatio
         children,
         className,
         link,
+        isLinkRef,
         target,
         showEmptyLink,
         [ "data-componentid" ]: componentId
     } = props;
 
+    const { getLink } = useDocumentation();
+    
     if (link === undefined) {
         return null;
     }
 
     const classes = classNames("documentation-link ml-1 link external no-wrap", className);
+    const url = isLinkRef? getLink(link): link;
```

Comment:
Missing space before the "?" and ":", should be 
`const url = isLinkRef ? getLink(link) : link;`

Reply:
Addressed via 6741675b2ae6fb0adaee3e1060740d33b8f171ed